cfme/networks/views.py

```diff
@@ -646,6 +646,34 @@ def is_displayed(self):
                 self.title.text == '{name} (Summary)'.format(name=self.context['object'].name))
 
 
+class ParentWithSubnetView(BaseLoggedInPage):
+    """ Represents Cloud Tenant's subnets page"""
+    title = Text('//div[@id="main-content"]//h1')
+    toolbar = View.nested(SubnetToolBar)
+    sidebar = View.nested(SubnetSideBar)
+    including_entities = View.include(SubnetEntities, use_parent=True)
+
+    @property
+    def is_displayed(self):
+        obj = self.context['object']
+        from cfme.networks.cloud_tenant import CloudTenant
+        from cfme.networks.network_router import NetworkRouter
+        is_tenant = isinstance(obj, CloudTenant)
+        is_router = isinstance(obj, NetworkRouter)
```

Comment:
Nitpicking, but we can inline this simple enough conversation (`if isinstance(obj, CloudTenant): ...`)